elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -138,29 +151,35 @@ async def _verify_attached_flaws_for(self, advisory_id: int, attached_trackers:
             else:
                 first_fix_flaw_ids = {
                     flaw_bug.id for flaw_bug in flaw_id_bugs.values()
-                    if bzutil.is_first_fix_any(self.bug_tracker.client(), flaw_bug, current_target_release)
+                    if bzutil.is_first_fix_any(self.runtime.bug_trackers('bugzilla'), flaw_bug, current_target_release)
                 }
 
         # Check if attached flaws match attached trackers
         attached_flaw_ids = {b.id for b in attached_flaws}
-        missing_flaw_ids = attached_flaw_ids - first_fix_flaw_ids
+        missing_flaw_ids = first_fix_flaw_ids - attached_flaw_ids
```

Comment:
I found this reversed for some reason @vfreex can you take a look I have it right?

Reply:
Yes, you are correct. Thanks for fixing this!